src/components/App/index.jsx

```diff
@@ -1,11 +1,14 @@
-import React, { Component } from 'react';
-
-import Header from '../Header/index.jsx';
-import Footer from '../Footer/index.jsx';
-import Main from '../Main/index.jsx';
+// import { connect } from 'react-redux';
+import React from 'react';
+import Header from '../../components/Header/index.jsx';
+import Footer from '../../components/Footer/index.jsx';
+import Main from '../../components/Main/index.jsx';
+// import setUser from '../../redux/actions/setUser.js';
 
 import './index.css';
 
+
+
 const App = props => {
```

Comment:
We can reduce that func component.
```suggestion
const App = () => (<Component />)
```